src/external-server/external-server.ts

```diff
@@ -86,17 +86,17 @@ export abstract class ExternalServer {
         this.cleanupFns.forEach((fn) => fn());
     }
 
-    protected abstract spawnChildProcess(): Promise<ChildProcess>;
+    protected abstract spawnChildProcess(options?: CommonSpawnOptions): Promise<ChildProcess>;
 
     protected abstract cleanup(): void;
 
-    public async start(): Promise<void> {
+    public async start(options?: CommonSpawnOptions): Promise<void> {
```

Comment:
I don't understand how this `options` object is used.

Reply:
https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/328/commits/12a77042e9f9527ca397075c1a58d0c813cc15f4